.github/workflows/main.yml

```diff
@@ -32,12 +32,15 @@ jobs:
       - name: Nuke Build 🏗
         id: build
         shell: bash
+        env:
+          REF: ${{ github.head_ref || github.ref }}
+          RUN_NUMBER: ${{ github.run_number }}
         # Pass branch and patch number to Nuke OctoVersion 
         # (for pull_request events we override the /refs/pull/xx/merge branch to the PR's head branch)
         run: |
           ./build.sh \
-            --OctoVersionBranch ${{ github.head_ref || github.ref }} \
-            --OctoVersionPatch ${{ github.run_number }}
+            --OctoVersionBranch $REF \
+            --OctoVersionPatch $RUN_NUMBER
```

Comment:
You could use octoversion env variables instead, see https://github.com/OctopusDeploy/LdapAuthenticationProvider/pull/92/files 🙂